src/scenes/ProgressReports/Detail/ProgressReportDrawer/Steps/PromptVariant/PromptsForm.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box, Typography } from '@mui/material';
+import { SubmissionErrors } from 'final-form';
+import { Form } from 'react-final-form';
+import { EnumField, EnumOption, SubmitButton } from '~/components/form';
+import { required } from '~/components/form/validators';
+import { RichTextView } from '~/components/RichText';
+import { ProgressReportAvailableDataFragment } from '../../ProgressReportDrawer.graphql';
+
+interface PromptsFormProps {
+  availableData: ProgressReportAvailableDataFragment | null;
+  reportId: string;
+  createItemMutation: (
+    input: any
+  ) => void | SubmissionErrors | Promise<SubmissionErrors>;
+}
+
+export const PromptsForm = ({
+  availableData: stepData,
+  reportId,
+  createItemMutation,
+}: PromptsFormProps) => {
+  return (
+    <Form
+      onSubmit={createItemMutation}
+      initialValues={{
+        reportId,
+      }}
+    >
+      {({ handleSubmit, validating }) => (
+        <form onSubmit={handleSubmit}>
+          <Box
+            sx={{
+              display: 'flex',
+              flexDirection: 'column',
+              padding: 2,
+              pt: 0,
+            }}
+          >
+            <Typography sx={{ mb: 2 }} variant="h3">
+              Share a team highlight story.
+            </Typography>
+            <Typography variant="body2" sx={{ mb: 2 }}>
+              As you reflect on the past three months, select ONE question to
+              answer.
+            </Typography>
+
+            {stepData?.prompts.length && (
+              <EnumField
+                name="prompt"
+                label="Select a question"
```

Comment:
We should be consistent with our noun. Since we talked this morning about using "Selected Prompt" in other view, we should use that noun here too.
```suggestion
                label="Select a prompt"
```